src/Raven.Client/Documents/Session/SessionOptions.cs

```diff
@@ -8,6 +8,12 @@ public enum TransactionMode
         ClusterWide,
     }
 
+    public enum ShardedBatchBehavior
+    {
+        SingleBucket,
```

Comment:
This means, error on a transaction that has multiple buckets (new behavior).
Right?